events/eamt-2024.md

```diff
@@ -1,75 +1,76 @@
 ---
+autogenerated: true
+autogenerated_from: events.json
 parent: Events
+nav_order: 2024
+layout: event
 title: EAMT 2024
+end_date: '2024-06-27'
+future_tense_opening_paragraph: The Conference of the European Association of Machine
+  Translation (<strong>EAMT 2024</strong>) will take place in Sheffield, England from
+  24 to 27 June, 2024.
+past_tense_opening_paragraph: The Conference of the European Association of Machine
+  Translation (<strong>EAMT 2024</strong>) took place in Sheffield, England from 24
+  to 27 June, 2024.
+name: EAMT 2024
+id: eamt-2024
 description: Conference of the European Association of Machine Translation
+calls_for_papers_deadline: '2024-03-08'
 location: Sheffield, England
-name: EAMT 2024
-startDate: 2024-06-24
-
+organizer:
+  type: Organization
+  name: European Association of Machine Translation
+  url: https://eamt.org
+start_date: '2024-06-24'
```

Comment:
I think we can print these without quotes.

Reply:
Only if it is easy - I know it is probably just the default in the conversion to YAML.